chain/types/ethtypes/eth_transactions.go

```diff
@@ -46,6 +46,7 @@ type EthTxArgs struct {
 	ChainID              int         `json:"chainId"`
 	Nonce                int         `json:"nonce"`
 	To                   *EthAddress `json:"to"`
+	From                 *EthAddress `json:"from"`
```

Comment:
I don't think we want to add this to `EthTxArgs` --the transaction arguments are independent of any single sender -- it should be the case that whoever signs them has a valid tx ready to go.